history-of-quakers/modernized/13-chapter-11.adoc

```diff
@@ -3078,8 +3183,16 @@ We do beseech you to tell all our dear friends, fathers, and elders,
 the pillars of our spiritual building, with all the rest of our Christian brethren,
 that we do desire their prayers, for we have need of them.
 
+--
+
+[.offset]
 This letter they concluded with the following Poem:
 
+[.embedded-content-document.letter]
+--
```

Comment:
I see why you did this as an embedded letter _and_ a [verse], (since it's a poem _in_ a letter) but I kind of lean towards not doing both, and just making it a verse, so why don't you go ahead and remove this, and it's ending.